docs/COMMIT_MESSAGES.md

```diff
@@ -15,8 +15,9 @@ Any commit needs a helpful message. Mind the following guidelines when committin
    - [transit]
    - [routing]
    - [bookmarks]
+   - [docs]
```

Comment:
[3party]

Reply:
surprisingly [3party] was never ever used! :)
and there is just one [submodules] entry